docs/integrations/azure/discovery.md

```diff
@@ -37,12 +90,6 @@ And then add the processors to your catalog builder:
 +  builder.addProcessor(AzureDevOpsDiscoveryProcessor.fromConfig(env.config, { logger: env.logger }));
 ```
 
-## Configuration
-
-To use the discovery processor, you'll need a Azure integration
-[set up](locations.md) with a `AZURE_TOKEN`. Then you can add a location target
```

Comment:
Isn't the token still needed? This should probably continue to be mentioned somewhere as I do run into this as an issue for people setting things up

Reply:
Re-read using outside of the PR in your branch and see it's still mentioned. I'm wondering if we should move the notes about the token, host and the note about search needing to be enabled or installed to the top of this page seeing as they are prerequisites for both this new provider and the existing processor. Could help save people time as it will be first thing they read

Reply:
Yep, that's a good ideal. I've reorganised the docs a bit, want to have another look?

Reply:
Love it, much nicer! Thanks for taking the time to make this change